packages/contentstack-export/src/export/modules-js/assets.js

```diff
@@ -327,8 +329,11 @@ module.exports = class ExportAssets {
   downloadAsset(asset) {
     const self = this;
     return new Promise(async (resolve, reject) => {
-      const assetFolderPath = path.resolve(self.assetsFolderPath, asset.uid);
-      const assetFilePath = path.resolve(assetFolderPath, asset.filename);
+      if(!validateUids(asset.uid) && !validateFileName(asset.filename)) {
+        reject(`UIDs not valid`)
+      }
+      const assetFolderPath = path.resolve(sanitizePath(self.assetsFolderPath), sanitizePath(asset.uid));
```

Comment:
@cs-raj , do we need to sanitize asset.uid?

Reply:
Yes, as we are passing it in path.resolve. Even though we are sure of the type of value it will contain